src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/SvDiscoverFromLocalAssemblyContigAlignmentsSpark.java

```diff
@@ -150,6 +151,7 @@ public boolean requiresReads() {
     protected void runTool(final JavaSparkContext ctx) {
 
         validateParams();
+        final Set<VCFHeaderLine> defaultToolVCFHeaderLines = getDefaultToolVCFHeaderLines();
```

Comment:
Seems like there's no reason to assign to a local variable, just pass the result of the call directly to SvDiscoveryInputMetadata()

Reply:
local var removed